style.css

```diff
@@ -0,0 +1,197 @@
+
+html {
+	font-size: 62.5%;
+}
+
+body {
+	font-size: 16px;
+	font-size: 1.6rem;
+	min-height: 100%;
+	overflow: hidden; /* Hide scrollbars */
+	background-color: #16191e;
+	color: #fdfdfd;
+	min-width: 300px;
+	min-width: 3.0;
+}
+
+/* Controls above table*/
+.controls {
+	display: flex;
+	text-overflow: clip; 
+    white-space: nowrap;
```

Comment:
Please change the spaces to tabs here. (You can add `"editor.renderWhitespace": "all",` to your vscode settings to catch stuff like this.)